pyodide-build/pyodide_build/testing.py

```diff
@@ -49,7 +49,7 @@ def run_in_pyodide(
         Whether to use a standalone selenium instance to run the test or not
     packages : List[str]
         List of packages to load before running the test
-    driver_timeout : Optional[Union[str, int]]
+    driver_timeout : Optional[int]
```

Comment:
This apparently wasn't supposed to support strings, it was being passed to selenium.

Reply:
Probably the best type here would be float or int. Thanks for catching it!

Reply:
Hmm, float would likely be better?

Reply:
If I have:
```py
def f(a : float): ...
```
and I call `f(5)` is that a type error? What about `n : int = 5; f(n)`?

Reply:
From [PEP 0484](https://www.python.org/dev/peps/pep-0484/#the-numeric-tower):

> Rather than requiring that users write import numbers and then use `numbers.Float` etc., this PEP proposes a straightforward shortcut that is almost as effective: when an argument is annotated as having type `float`, an argument of type `int` is acceptable;